test/test.sh

```diff
@@ -6,6 +6,7 @@
 
 # Don't assume the test is being run from the same directory as the script
 TEST_DIR="$(dirname "$0")"
+WGETPASTE="${TEST_DIR}/../build/image/bin/wgetpaste"
```

Comment:
This will break if someone uses a different build folder name. I'd registering the tests using Meson's test functionality (see https://mesonbuild.com/Unit-tests.html). Afterwards, it should be possible to make it dynamic:

```suggestion
WGETPASTE="${MESON_BUILD_ROOT}/bin/wgetpaste"
```

Reply:
done